app/models/health_care_application.rb

```diff
@@ -287,13 +287,13 @@ def form_matches_schema
   end
 
   def current_schema
-    schema = VetsJsonSchema::SCHEMAS[self.class::FORM_ID]
-    return schema unless Flipper.enabled?(:hca_use_facilities_API)
-
     Rails.logger.warn(
-      "HealthCareApplication::hca_use_facilitiesAPI enabled = #{Flipper.enabled?(:hca_use_facilities_API)}"
+      "HealthCareApplication::hca_use_facilitiesAPI enabled = #{Flipper.enabled?(:hca_use_facilities_API, user)}"
     )
 
+    schema = VetsJsonSchema::SCHEMAS[self.class::FORM_ID]
+    return schema unless Flipper.enabled?(:hca_use_facilities_API, user)
```

Comment:
I don't think we are properly testing this line. I commented it out and ran the model and request specs and everything passes 🤔 .

Reply:
I think feature flags might be on by default in the specs after all? adding an explicitly-disabled check now.